projects/probational/cmd/probational.go

```diff
@@ -0,0 +1,249 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Moby is the type of a Moby config file
+// stolen from github.com/moby/tool/cmd/moby, perhaps we should make handling
+// this into a lib?
+type Moby struct {
+	Kernel struct {
+		Image   string
+		Cmdline string
+	}
+	Init []string
+	//Onboot   []MobyImage
+	//Services []MobyImage
+	//Trust    TrustConfig
+	Files []struct {
+		Path      string
+		Directory bool
+		Symlink   string
+		Contents  string
+		Source    string
+	}
+}
+
+const linuxkitPath = "../linuxkit.yml"
+const kernelSeries = "4.11.x"
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Printf("error: %s", err)
+		os.Exit(1)
+	}
+}
+
+func run() error {
+	var projectsFile string
+
+	flag.StringVar(&projectsFile, "input", "probational/projects.yml", "The input list of projects")
+	flag.Parse()
+
+	content, err := ioutil.ReadFile(projectsFile)
+	if err != nil {
+		return err
+	}
+
+	var projects []string
+	if err := yaml.Unmarshal(content, &projects); err != nil {
+		return err
+	}
+
+	content, err = ioutil.ReadFile(linuxkitPath)
+	if err != nil {
+		return err
+	}
+
+	os.Remove("kernel-config/kernel_config.probational")
```

Comment:
Because it's generated, and we always append to it. I suppose we could remove it as part of the initial build process too.